terraform/main.tf

```diff
@@ -33,10 +33,18 @@ module "lambdaedge_function" {
   config = var.environment
 }
 
+module "waf_rules" {
+  source = "./modules/waf_rules"
+  providers = {
+    aws = aws.us
```

Comment:
aws.us should be more detailed imho.
just in case you'd wanted to address the exact region-(code) as well

Reply:
fixed.